src/gui/Measurement.h

```diff
@@ -0,0 +1,21 @@
+
+#pragma once
+
+#include <QGLView.h>
```

Comment:
```suggestion
#include "QGLView.h"
```

Naming of that file is a bit unfortunate, it's not a Qt header but belongs to OpenSCAD.